docs/security.md

```diff
@@ -57,17 +59,27 @@ presetUsers:
     privileges: "lb_api"
 ```
 
-Provide a key pair ins RSA format
+Also provide a random key pair in RSA format.
 
 ```
+authentication:
+  defaultType: "form"
   form:
     selfSignKeyPair:
-      privateKeyRSA: |
-      <PEM PRIVATE KEY>
-      publicKeyRSA: |
-      <PEM PUBLIC key>
+      privateKeyRsa: <private_key_path>
+      publicKeyRsa: <public_key_path>
 ```
 
+### Form/LDAP
```

Comment:
I would merge this section with the above so it becomes 

```
  authentication:
    defaultType: "oauth"
    form:
      ldapConfigPath: <ldap_config_path>
      selfSignKeyPair:
        publicKeyRsa: "<public_key_path>"
        privateKeyRsa: "<private_key_path>"
```
Since `selfSignKeyPair` and `ldapConfigPath` are both needed by `FormAuthConfiguration`

Reply:
Is both `ldapConfigPath` and `selfSignKeyPair` needed for ldap? because ldap config can be left empty if we do use FormAuthConfiguration but not use ldap

Reply:
I think its fine to have them separate and explain each section .. and then also maybe have examples that combine each. We can iterate on that over time.

Reply:
@Chaho12 I think both `ldapConfigPath` and `selfSignKeyPair` are needed if we have `defaultType = form` for authentication.

Reply:
> if we have defaultType = form for authentication.

For both ldap and basic auth? 
With current main version, we are using the conf as below and it works for admin/user1 to login and blocks any other users. 
```yaml
authentication:
      defaultType: "form"
      form:
        selfSignKeyPair:
          privateKeyRsa: /app/config/private.key
          publicKeyRsa: /app/config/public.key
presetUsers:
      admin:
        password: ...
        privileges: "lb_admin, lb_user, lb_api"
      user1:
        password: ...
        privileges: "lb_user"
    authorization:
      admin: "lb_admin"
      user: "lb_user"
      api: "lb_api"
```

Reply:
oh I see. In this case we don't need to have both then.

Reply:
So I think this PR is ready to go then.